docs/docs/usage/dependency.md

```diff
@@ -136,34 +136,50 @@ To remove existing dependencies from project file and the library directory:
 ```console
 # Remove requests from the default dependencies
 $ pdm remove requests
-# Remove h11 from the 'web' optional group
+# Remove h11 from the 'web' section of optional-dependencies
 $ pdm remove -s web h11
-# Remove pytest-cov from the `test` dev-dependencies group
+# Remove pytest-cov from the `test` section of dev-dependencies
 $ pdm remove -ds test pytest-cov
 ```
 
 ## Install the packages pinned in lock file
 
 There are two similar commands to do this job with a slight difference:
 
-```console
-# Install all default dependencies
-$ pdm install
-# Install default + web optional dependencies
-$ pdm install -s web
-# Install default + all optional dependencies
-$ pdm install -s:all
-# Install default + all dev-dependencies
-$ pdm install -d
-$ pdm install -ds:all
-# Install web dependencies ONLY (without default dependencies)
-$ pdm install --no-default -s web
-```
-
 - `pdm install` will check the lock file and relock if it mismatches with project file, then install.
 - `pdm sync` installs dependencies in the lock file and will error out if it doesn't exist.
   Besides, `pdm sync` can also remove unneeded packages if `--clean` option is given.
 
+### Select a subset of dependencies with CLI options
+
+Say we have a project with following dependencies:
+
+```toml
+[project]  # This is production dependencies
+dependencies = ["requests"]
+
+[project.optional-dependencies]  # This is optional dependencies
+extra1 = ["flask"]
+extra2 = ["django"]
+
+[tool.pdm.dev-dependencies]  # This is dev dependencies
+dev1 = ["pytest"]
+dev2 = ["mkdocs"]
+```
+
+| Command                         | What it does                                                             | Comments                  |
+| ------------------------------- | ------------------------------------------------------------------------ | ------------------------- |
+| `pdm install`                   | install prod and dev deps(no optional)                                   |                           |
+| `pdm install -s extra1`         | install prod deps, dev deps, and "extra1" optional section               |                           |
+| `pdm install -s dev1`           | install prod deps and only "dev1" dev section                            |                           |
+| `pdm install -s:all`            | install prod deps and "extra1", "extra2" optional section                |                           |
```

Comment:
Shouldn't this one also install all dev dependencies?

Reply:
Oh year, you are right